torch/csrc/jit/passes/python_print.cpp

```diff
@@ -713,7 +713,7 @@ struct PythonPrintPass {
         if (enforce_importable_) {
           throw script::ErrorReport(node->getSourceLocation())
               << "could not export python function call " << value->name()
-              << ". Remove calls to Python functions before export";
+              << ". Remove calls to Python functions before export (did you forget to add it to __constants__?)";
```

Comment:
There should be some filtering here, if this printed when people include a Python function in their model and then export, this suggests adding it to `__constants__` will fix it

Reply:
I'm not sure how to filter that here